flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/Schema.java

```diff
@@ -620,15 +631,27 @@ private void addResolvedConstraint(UniqueConstraint constraint) {
     /** Super class for all kinds of columns in an unresolved schema. */
     public abstract static class UnresolvedColumn {
         final String columnName;
+        final @Nullable String comment;
 
         UnresolvedColumn(String columnName) {
```

Comment:
@wuchong  I found this constructor is never used now , should I  remove this ? .

Reply:
Sure.